### DISCUSSION TITLE
Where to find information to update /helix/language.toml

### DISCUSSION THREAD

**Discussion by lauccode** [helix-editor/helix#3979] [Q&A]
I was looking to use helix for cpp with vscode-cpptools.
But I can not find the way to update the /helix/language.toml.
I tested it with lunarvim. And it is ok. 

I guess it could work with helix now just be updating the .toml.
Should be nice to know how to update .toml easily ?

I can see the rust example in helix documentation. 
But I don't know how to set exactly the good value to set cpp debugger with vscode-cpptools ?

**Comment by the-mikedavis** [comment]
The `languages.toml` file is here: https://github.com/helix-editor/helix/blob/master/languages.toml but you can also have one in your `~/.config/helix/languages.toml` which overrides keys from the one in this repository. For example if I wanted to disable auto-format in rust, I could use:

```toml
# ~/.config/helix/languages.toml
[[language]]
name = "rust"
auto-format = false
```

The current configuration for CPP files is here: https://github.com/helix-editor/helix/blob/e8f0886b21d279fedce53d85f53bb537c1ad808e/languages.toml#L189-L220. Also see https://docs.helix-editor.com/languages.html but currently DAP related configuration and DAP usage are experimental and undocumented.

**Comment by lauccode** [comment]
Hello, thanks a lot for all information.
I guess it means that I need to wait documented usage before to try to use vscode-cpptools ?

I'm looking to open automatically files when I open helix rather than open it one by one each time.
This is the main point I don't start to use helix now.
Some features are missing. But it seems to be enough to start to edit at least.

**Comment by lauccode** [comment]
I found a workaround to manage project with git(with help of lazygit) to be able to open multiple files of a commit with **_Helix_** :
https://github.com/jesseduffield/lazygit/issues/2187
